docs/reference/target-frameworks.md

```diff
@@ -78,13 +78,15 @@ Universal Windows Platform | uap | uap [uap10.0] |
 | | | netstandard1.6 |
 | | | netstandard2.0 |
 | | | netstandard2.1 |
-.NET Core App | netcoreapp | netcoreapp1.0 |
+.NET 5+ (and .NET Core) | netcoreapp | netcoreapp1.0 |
 | | | netcoreapp1.1 |
 | | | netcoreapp2.0 |
 | | | netcoreapp2.1 |
 | | | netcoreapp2.2 |
 | | | netcoreapp3.0 |
 | | | netcoreapp3.1 |
+| | net | net5.0 |
+| | net | net6.0 |
```

Comment:
remove this `net` to be consistent & we'll merge!

Reply:
Sorry just one of them! 

![image](https://user-images.githubusercontent.com/1288848/115595745-e7a6d380-a29c-11eb-9dc4-a612eaa7f158.png)

Reply:
you can keep the `net` on `net5.0`! It looks great there.

Reply:
> you can keep the `net` on `net5.0`! It looks great there.
agreed.